Mathlib/Data/List/Infix.lean

```diff
@@ -469,9 +469,9 @@ theorem insert_nil (a : α) : insert a nil = [a] :=
   rfl
 #align list.insert_nil List.insert_nil
 
-theorem insert.def (a : α) (l : List α) : insert a l = if a ∈ l then l else a :: l :=
+theorem insert_eq_ite (a : α) (l : List α) : insert a l = if l.elem a then l else a :: l :=
```

Comment:
Can do keep the old statement? I don't think any mathlib user is going to want this one.

Reply:
No user wants either, apparently, which is why I deprecated it.

Reply:
I've changed the statement now.